blockreassurance.php

```diff
@@ -493,6 +498,9 @@ private function renderTemplateInHook($template)
         return $this->display(__FILE__, 'views/templates/hook/' . $template);
     }
 
+    /**
+     * @throws PrestaShopException
```

Comment:
I dont know if it's necessary to put all these `@throws PrestaShopException` 🤔 

Just a suggestion, nothing mandatory

Reply:
That permits to PHPStorm to not have warnings on these lines. :ok: ?

Reply:
All right 👍